ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityAdapter.java

```diff
@@ -274,20 +218,63 @@ public void showInterstitial() {
       return;
     }
 
-    if (!UnityAds.isReady(mPlacementId)) {
+    if (mLoadedPlacementId == null) {
```

Comment:
See my mopub PR here:https://github.com/Applifier/mopub-android-mediation/pull/6
You probably want to still call show (though likely with mPlacementId since that is what was attempted to be loaded) and let the show call fail with appropriate response.  if mPlacement is null then load was never called likely.

Reply:
We may want to keep this check here for the same reason as here:
https://github.com/Applifier/mopub-ios-mediation/pull/10/files#r601805396

Or remove the both.